lib/spack/spack/cmd/config.py

```diff
@@ -68,6 +68,18 @@ def setup_parser(subparser):
 
     sp.add_parser("list", help="list configuration sections")
 
+    list_scopes_parser = sp.add_parser("list-scopes", help="list defined scopes")
+    stype = list_scopes_parser.add_mutually_exclusive_group(required=False)
```

Comment:
I know there aren't a lot of scopes by default, but why are these mutually exclusive?

Reply:
I think because the way I was originally obtaining non-platform scopes, they were `--file` by construction.